xml/admin_ceph_gateway.xml

```diff
@@ -2879,8 +2879,14 @@ rgw_override_bucket_index_max_shards = 12
     <para>
      On the &ogw; host, start and enable the &cogw; service:
     </para>
-<screen>&prompt.cephuser.ogw;systemctl start ceph-radosgw@rgw.`hostname -s`
-&prompt.cephuser.ogw;systemctl enable ceph-radosgw@rgw.`hostname -s`</screen>
+<screen>&prompt.cephuser.ogw;systemctl start ceph-<replaceable>FSID</replaceable>@<replaceable>DAEMON_NAME</replaceable>
+&prompt.cephuser.ogw;systemctl enable ceph-<replaceable>FSID</replaceable>@<replaceable>DAEMON_NAME</replaceable></screen>
+   <note>
+     <para>
+       To ensure you have the relevant daemon name, run the following command:
```

Comment:
@tbazant do you think we should include this note everywhere?

Reply:
@asettle I would make this note a tip and prepend it before the `systemctl` commands, saying something like "To identify the unique FSID of the cluster, run `ceph fsid`. To identify the &ogw; daemon name, run `ceph orch ps ---hostname &lt;replaceable>HOSTNAME&lt;/replaceable>`"

Reply:
my second thought is that we have a chapter about operating Ceph services, so....i would just refer it not to cause too much noise and mess
see https://susedoc.github.io/doc-ses/master/single-html/ses-admin/#ceph-operating-services-finding-names

Reply:
@sebastian-philipp either of those options suit you?

Reply:
I honestly don't mind. whatever you thing is better.